src/layouts/home/user-list-item/utils.ts

```diff
@@ -1,9 +1,12 @@
-import { formatDistanceStrict } from 'date-fns';
+import { ChatUserBriefResponse } from '@api/lib';
 
-export const getDateDistance = (date: Date | string | number) => {
-  const time = formatDistanceStrict(new Date(date), new Date());
-  if (time.includes('seconds') && parseInt(time) <= 30) {
-    return 'Now';
-  }
-  return time;
-};
+export const avatarStyles = 'mr-2.5 w-10 h-10 min-w-10 min-h-10';
```

Comment:
Why did you move the style constant to a separate file? Move it to the usage file, but after the imports